issue #55 - added additional status bar updates and running/error ind…

bfitzpat opened the pull request:
…icators in the integrations view 
-added new setting to turn off status bar messages 
-fixed small issue with message on follow log error

Signed-off-by: Brian Fitzpatrick <bfitzpat@redhat.com>

bfitzpat commented on src/CamelKNodeProvider.ts:
```diff
@@ -248,17 +257,38 @@ export class CamelKNodeProvider implements vscode.TreeDataProvider<TreeNode> {
+		this.status = status;
+		this.iconPath = this.getIconForPodStatus(this.status);
+		this.tooltip = `Status: ${this.status}`;
+		//  this.iconPath = {
```
Will clean it up. No longer needed.

apupier commented:
(argh, I lost some parts of my review, i forgot to submit it)

- is the image without dot still used somewhere? if not, might be good to remove.

bfitzpat commented:
> (argh, I lost some parts of my review, i forgot to submit it)
>     * is the image without dot still used somewhere? if not, might be good to remove.

No, but I wanted to keep it in case we needed a base image to add additional decorations to down the line.

apupier commented:
> > * is the image without dot still used somewhere? if not, might be good to remove.
> 
> No, but I wanted to keep it in case we needed a base image to add additional decorations to down the line.

ok

apupier commented:
thinking of it only now, but would worth updating the images in the readme too . i think only this one is affected: https://github.com/camel-tooling/vscode-camelk/blob/master/images/camelk-integrations-view-refresh-action.jpg

apupier commented on src/CamelKNodeProvider.ts:
```diff
@@ -248,17 +257,34 @@ export class CamelKNodeProvider implements vscode.TreeDataProvider<TreeNode> {
+		let newIcon : object;
+		if (status.toLowerCase().startsWith("running")) {
+			newIcon = { 
+				light: vscode.Uri.file(path.join(__dirname, "../resources/round-k-transparent-16-running.svg")),
```
light and dark theme are using the same image? Maybe we can get rid of the 2 cases then?

bfitzpat commented:
> thinking of it only now, but would worth updating the images in the readme too

Several images were affected actually, so this was a great reminder. Thanks. Updated and added new details around updated settings, status bar messages, and so on.
